hutch_python/tests/test_log_setup.py

```diff
@@ -197,3 +198,55 @@ def find_filters():
         "shown-message-2",
         "No longer recording log messages from obj",
     }
+
+
+@pytest.fixture(scope="function")
+def object_filter(
+    monkeypatch, log_queue: queue.Queue
+) -> log_setup.ObjectFilter:
+    def no_op(*args, **kwargs):
+        ...
+
+    monkeypatch.setattr(log_setup.ObjectFilter, "_count_update_thread", no_op)
```

Comment:
This was a really good solution to taking the timer out of the equation for the test suite. You've essentially taken control of the clock and can advance it yourself whenever you want.

Reply:
My first inclination was to add a `start` kwarg to ObjectFilter, further complicating it. I think changing it to this was a win all-around 👍